### PR TITLE
Set Radio selection vertically for callout on condition

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
Change the orientation of Radio buttons in Callout component depending on the number of radio inputs

### Before
Screenshot 2020-11-11 at 12 40 18

### After
2 Radio
Screenshot 2020-11-11 at 13 24 20

More than 2 Radio
Screenshot 2020-11-11 at 12 39 51

## Why?
As mentioned by @SiAdcock https://github.com/guardian/source/issues/580#issuecomment-718728816, it is recommended  to use horizontal radio buttons if there are only 2 answers.

### CODE REVIEW

File: src/web/components/Callout/RadioSelect.tsx [hunk 1]
```diff
@@ -25,7 +25,12 @@ export const RadioSelect = ({
         `}
     >
         <FieldLabel formField={formField} />
-        <RadioGroup name={formField.name} orientation="horizontal">
+        <RadioGroup
+            name={formField.name}
+            orientation={
+                formField.options.length > 2 ? 'vertical' : 'horizontal'
```
Comment (oliverlloyd) [inline src/web/components/Callout/RadioSelect.tsx:31]: 👍

**Review by oliverlloyd — APPROVED** [review]

**Comment by oliverlloyd** [comment]
In the Chromatic snapshot it looks like the radio field with 2 options is not showing horizontally

Screenshot 2020-11-11 at 13 39 50

Is that desired?

**Comment by liywjl** [comment]
> In the Chromatic snapshot it looks like the radio field with 2 options is not showing horizontally
> 
> Screenshot 2020-11-11 at 13 39 50
> 
> Is that desired?

Yes so that is an interesting point, this is because Chromatic uses an older version of Chrome that has a bug in `flex` https://github.com/guardian/source/issues/580 , the bug has since been fixed

**Comment by oliverlloyd** [comment]
> > In the Chromatic snapshot it looks like the radio field with 2 options is not showing horizontally
> > Screenshot 2020-11-11 at 13 39 50
> > Is that desired?
> 
> Yes so that is an interesting point, this is because Chromatic uses an older version of Chrome that has a bug in `flex` [guardian/source#580](https://github.com/guardian/source/issues/580) , the bug has since been fixed

Wild. But out of our control so 👍

**Pull request merged**